frontend/src/features/admin-form/responses/IndividualResponsePage/PaymentSection.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Divider, Link, Stack, Text } from '@chakra-ui/react'
+import { keyBy } from 'lodash'
+
+import { SubmissionPaymentDto } from '~shared/types'
+
+import { getPaymentDataView } from '../common/utils/getPaymentDataView'
+
+const PaymentDataItem = ({
+  name,
+  value,
+  isMonospace,
+  isUrl,
+}: {
+  name: string
+  value: string
+  isMonospace?: boolean
+  isUrl?: boolean
+}) => (
+  <Stack direction={{ base: 'column', md: 'row' }}>
+    <Text textStyle="subhead-1">{name}:</Text>
+    <Text
+      sx={
+        isMonospace
+          ? {
+              fontFeatureSettings: "'tnum' on, 'lnum' on, 'zero' on, 'cv05' on",
```

Comment:
Does it make sense to move this into `textStyle` as `monospace`? It seemed to be a generic style for monospaced text.

Reply:
good point, have moved it!